breathecode/registry/views.py

```diff
@@ -520,6 +520,10 @@ def get(self, request, asset_slug=None):
         if need_translation == 'true':
             items = items.annotate(num_translations=Count('all_translations')).filter(num_translations__lte=1)
 
+        if 'category_not' in self.request.GET:
```

Comment:
`exclude_category`

Reply:
Yo opinaba poner este nombre tambien pero alejandrome dijo que pusiera category not, que se ve raro que no empiece igual por category